docs/pages/includes/database-access/aws-troubleshooting-max-policy-size.mdx

```diff
@@ -0,0 +1,61 @@
+### Maximum policy size exceeded errors
+
+Due to [IAM and STS character
+limits](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_iam-quotas.html#reference_iam-quotas-entity-length),
+you may encounter one of the following errors in the Database Service logs, when large numbers of databases are
+registered:
```

Comment:
Removed comma, fixed hard breaks:
```suggestion
you may encounter one of the following errors in the Database Service logs when
large numbers of databases are registered:
```

Reply:
I had a 120 textwidth for some other changes and forgot to reset it to 80. Adjusted all paragraphs to 80 now. Thanks!